src/astro/databases/google/bigquery.py

```diff
@@ -49,6 +49,12 @@ class BigqueryDatabase(BaseDatabase):
     NATIVE_PATHS = {
         FileLocation.GS: "load_gs_file_to_bigquery",
         FileLocation.S3: "load_s3_file_to_bigquery",
+        FileLocation.LOCAL: "load_local_file_to_bigquery",
+    }
+    NATIVE_PATHS_SUPPORTED_FILE_TYPES = {
+        FileType.CSV: "CSV",
+        FileType.NDJSON: "NEWLINE_DELIMITED_JSON",
```

Comment:
Maybe we should make these members of the parent class? It feels weird to have to redefine them for each class

Reply:
@dimberman  It's a little weird that they require to pass a different value for NDJSON file types. That's why I maintained two different dicts. I cleared this a bit, hope you like it. :)